sw/device/lib/dif/BUILD

```diff
@@ -0,0 +1,34 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+cc_library(
+  name = "clkmgr",
+  srcs = [
+      ":dif_clkmgr.c",
+      ":dif_clkmgr.h",
+      ":dif_warn_unused_result.h",
+      ],
```

Comment:
The `srcs` list should not include the leading colon `:` character.  We're listing files, not providing bazel labels.

The file `dif_clkmgr.h` should be listed in `hdrs` because it is the public interface to dif_clkmgr (if, OTOH, it was a private header file no dependent would be expected to include, _then_ it would belong in `srcs`).

Reply:
Oh, I wonder if that fixes anything else. I'll make the fix. I was wondering why bazel distinguished between headers, dependencies and sources.